pilot/lib/pilot/tester_manager.rb

```diff
@@ -55,8 +56,10 @@ def find_tester(options)
     def remove_tester(options)
       start(options)
 
-      tester = Spaceship::Tunes::Tester::External.find(config[:email])
-      tester ||= Spaceship::Tunes::Tester::Internal.find(config[:email])
+      tester = Spaceship::Tunes::Tester::Internal.find(config[:email])
+      UI.user_error!("#{tester.email} is an internal tester; pilot does not support internal testers") unless tester.nil?
+
+      tester = Spaceship::Tunes::Tester::Internal.find(config[:email])
```

Comment:
Sure! Good thought - don't make that extra network request (to look for internal tester) in the happy path.